src/lib/Microsoft.Health.Common/SimpleAggregateException.cs

```diff
@@ -0,0 +1,54 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using EnsureThat;
+
+namespace Microsoft.Health.Common
+{
+    public class SimpleAggregateException : Exception
```

Comment:
Would be good to double check if we don't have any thing in the health care shared components that does something similar.  If not this might be a good time to add a dependency to that library and add this there so it can be reused, https://github.com/microsoft/healthcare-shared-components

Reply:
There is nothing similar in the shared components. But since we'll be logging the individual exceptions anyway this class doesn't seem to be needed.